website/www/site/content/en/documentation/dsls/sql/overview.md

```diff
@@ -23,13 +23,11 @@ bounded and unbounded `PCollections` with SQL statements. Your SQL query
 is translated to a `PTransform`, an encapsulated segment of a Beam pipeline.
 You can freely mix SQL `PTransforms` and other `PTransforms` in your pipeline.
 
-Beam SQL includes the following dialects:
+Beam SQL uses Calcite SQL based on [Apache Calcite](https://calcite.apache.org),
+a dialect widespread in big data processing.
 
-- [Beam Calcite SQL](https://calcite.apache.org)
-- [Beam ZetaSQL](https://github.com/google/zetasql)
-
-Beam Calcite SQL is a variant of Apache Calcite, a dialect widespread in
-big data processing. Beam Calcite SQL is the default Beam SQL dialect. Beam ZetaSQL is more compatible with BigQuery, so it's especially useful in pipelines that [write to or read from BigQuery tables](https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.html).
+<!-- TODO: Remove deprecation statement when ZetaSQL dialect is removed. -->
+**Note:** Beam SQL supports for [ZetaSQL dialect]((/documentation/dsls/sql/zetasql/overview) has been deprecated.
```

Comment:
I would also point them to the calcite documentation (https://github.com/apache/beam/pull/34563/files#diff-6a9957943bb5546ab241109acc0122825b9fc405b47ff78b1a7b3d08a0103735L49)